cfme/fixtures/vm.py

```diff
@@ -11,7 +12,15 @@
 
 def _create_vm(request, template, provider, vm_name):
     collection = provider.appliance.provider_based_collection(provider)
-    vm_obj = collection.instantiate(vm_name, provider, template_name=template.name)
+    try:
+        assignee = DocstringParser(request.function.__doc__).parse()["assignee"].value
```

Comment:
With the most recent versions of the polarion tools (PR #9225 includes update) this metadata (composited from module/class/function) is available from the session.

Currently that session attribute is only populated if in collect only mode, I'm looking to change that in pytest-polarion-collect PR 5 or a followup.

This works for now for testing this functionality and designing the fixture structure, so I don't think it blocks working the rest of this PR.  But ASAP this should be changed to use the pre-parsed (and composited) metadata.

Reply:
I have access to items and fspath through the session object. I do see nodeid too but that returns blank. I don't see what we can use to fetch the metadata as you mentioned in the comment above. I agree that once it becomes available we can use it. 
```
n [15]: s.items                                                                                                                                                                                                                               
Out[15]: [<Function 'test_vm_name_postfix[virtualcenter-6.7-full_template]'>]

In [16]: s.fspath                                                                                                                                                                                                                              
Out[16]: local('/home/kkulkarn/git/integration_tests')

In [17]: s.nodeid                                                                                                                                                                                                                              
Out[17]: ''
```